### ISSUE TITLE
Clade constraints and members - discussion thread

### ISSUE THREAD

**Issue by PaulHeggarty** [lingdb/CoBL#122]

**Comment by PaulHeggarty** [comment]
Should Old Church Slavonic be forced as specifically South Slavic?  Perhaps best just let the data decide.

**Comment by leszekjocz** [comment]
OCS (or, to be more precise, its Bulgar-Macedonian base) has clear dialectal features and a clear dialectal affiliation. Among others, it has some important South Slavic phonetic and morphological features.

**Comment by m-scarborough** [comment]
I see that the new branch lists (which look great, by the way) are putting Greek and Armenian together for comparison.  I'm assuming this is for presentation purposes and not implying Greco-Armenian as a clade constraint, since I believe that the majority of Indo-Europeanists would not subgroup these two branches as a clade without some careful reservations (similar to the vexed question of Italo-Celtic).

I wonder if these should be split into separate Greek and Armenian lists in the branch lists to avoid implying that the data is taking Greco-Armenian as a prior?

**Comment by morenovuleta** [comment]
Well, since the overwhelming majority of cited forms comes from canonical texts, it would do justice to classify it within South Slavic, however, as the idea of the project is to compare them using information science techniques, it makes even more sense to see what a computer may tell us

**Comment by PaulHeggarty** [comment]
Matthew, not sure what you mean:  which branch lists put Greek and Armenian together.  It is definitely something we do not want, and in the branches table (click the tree icon, or http://www.cobl.info/languageBranches/ ), Greek is 5-0 and Armenian is 9-3

**Comment by m-scarborough** [comment]
Ah ok, I see now that Greek and Armenian are definitely separate in the assumed priors.  That was a misunderstanding on my part.  I do still think, however, that grouping them together in the language list http://cobl.info/languagelist/01_Greek_and_Armenian/ may be slightly misleading to any non-Indo-Europeanists viewing the database.

**Comment by PaulHeggarty** [comment]
Good point.  Those lists are really just for convenience, and are in part superseded now that we have real filtering by branch working (with a last few tweaks still needed).  But I agree that it's open to misinterpretation, so I think we should indeed split up these lists -- Cormac?

**Comment by PaulHeggarty** [comment]
Thanks for the answers on OCS as definitely to go within South Slavic.

**Comment by CasFre** [comment]
As for the Graeco-Armenian list: If it does not take too much time, I would also say they should be split up. In that list-menu we even have Iranian and Indic separated, so on that level Greek and Armenian should be too.

**Comment by CasFre** [comment]
@PaulHeggarty: Did you put up the files I created anywhere? If we already have this discussion thread, it might be worth publishing those trees so that the branch specialists and others interested may have a look at it and give us their feedback before we settle on anything.

**Comment by cormacanderson** [comment]
I can split up the Greek_and_Armenian list, although those lists are really just there for the convenience of people putting in data, so that there are less languages to trawl through. It was not named Graeco-Armenian specifically to avoid making any claims about it, but it is no problem to split them up and I will do that later today (have to do some work on the language lists anyway). As @PaulHeggarty says, the branch filtering should render those lists largely obsolete anyway.

**Comment by PaulHeggarty** [comment]
They will be obsolete except for the export function, if we want to do focused, faster tests on particular branch hypotheses, such as Greco-Armenian, of course.  I suspect that this means that once filtering is fully working, we should switch to a public view without such sublists, and keep these list in reserve only for editors to do focused exports.
